src/module/settings.ts

```diff
@@ -29,6 +30,15 @@ export const registerSettings = function () {
     type: Object,
   });
 
+  game.settings.registerMenu(LANCER.sys_name, "compconLogin", {
+    name: "Comp/Con Login",
+    label: "Log in to Comp/Con",
+    hint: "Log in to Comp/Con to automatically load any pilots and mechs you have access to",
+    icon: "fas fa-bars",
+    type: CompconLoginForm,
+    restricted: true
```

Comment:
Since the data storage is all local, I think all users should have access to this. Probably need to make the setting either user-level or client-level, testing now.